go/oasis-node/cmd/debug/txsource/workload/workload.go

```diff
@@ -0,0 +1,16 @@
+package workload
+
+import (
+	"math/rand"
+
+	"google.golang.org/grpc"
+
+	consensus "github.com/oasislabs/oasis-core/go/consensus/api"
+	runtimeClient "github.com/oasislabs/oasis-core/go/runtime/client/api"
+)
+
+type Runner func(rng *rand.Rand, conn *grpc.ClientConn, cnsc consensus.ClientBackend, _ runtimeClient.RuntimeClient) error
```

Comment:
Consider making this an interface.

Reply:
ok. it's more enterprise that way anyway